turingschool-examples/test-express
# Update dependencies & include polyfill and cors

#### How to test?

- Install dependencies and run `npm start`.  It should run with no issues.  Can go to localhost:3000 and get a response.
- Try running migrations and seed data locally.
- Run `npm t` and it should have one passing test.

@thatPamIAm Let me know if this works for you locally on your machine.  Thank you!
